.github/actions/prepare-workspace/action.yml

```diff
@@ -0,0 +1,16 @@
+name: "Prepare Node.js workspace"
+description: "Prepares Node.js workspace with installed dependencies"
+
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 20
```

Comment:
There is `.nvmrc` file specifying Node v18. There is an option to configure `node-version-file: '.nvmrc'`, so that the version is specified in one place.

Reply:
cool, didn't know about this option, thanks, fixed :)